client/src/components/GoBackButton/GoBackButton.module.scss

```
@@ -1,5 +1,7 @@
 @import "../../global.scss";
 
+// ACHTUNG: da es jetzt ein Button ist, müssen Background und border auf 'none' gesetzt werden. Beim styling bitte erneut anpassen"
+
 .goBackButton {
     width: 2.5vw;
     height: 4vh;
```

Das kann ich für dich machen :D
```suggestion
// ACHTUNG: da es jetzt ein Button ist, müssen Background und border auf 'none' gesetzt werden. Beim styling bitte erneut anpassen"

.goBackButton {
    width: 2.5vw;
    height: 4vh;
    
    background: none;
    border: none;
```

---

komplett neu gestylt inzwischen!